Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM emscripten/emsdk:1.39.18
+
```

Comment:
Could you add  "USER 1000" here? Otherwise all files created in the mounted folder are owned by root.